src/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -30,7 +30,7 @@ The HTML code for the custom page must meet the following requirements:
 -  Specify all CSS inline in the HTML source.
 -  Bundle images in the HTML page using base64 so that they display even if Fastly is offline. See [Data URIs on the css-tricks site](https://css-tricks.com/data-uris/).
 
-## Customize the 503 error page
```

Comment:
There's at least one link in this document that goes to `(#customize-the-503-error-page)` (line 23). We need to make sure that we don't introduce any broken links with this change. 

Thank you

Reply:
Golden eye, @rogyar !

**fixed** and **verified** via local preview.

Thank you!